docs/content/community/getting-help.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Getting Help | Dagster
+description: Have questions about how to use Dagster? Hit a bug? Have a feature request? This page includes tips on what to do.
+---
+
+# Guide to Getting Help
```

Comment:
```suggestion
# Getting help
```

Thoughts on removing `Guide to`? Also suggesting we use sentence case here to be consistent with our style guide